main.tf

```diff
@@ -1,96 +0,0 @@
-/**
```

Comment:
I'm a bit surprised by the fact that this module doesn't have a `main.tf`, since it's a common convention. Is the equivalent logic broken up into `client.tf` and `server.tf`?

Reply:
Yes